src/style.css

```diff
@@ -1,3 +1,74 @@
+@font-face {
+  font-family: Roboto;
+  src: url(fonts/Roboto/Roboto-Medium.ttf);
+}
+
 body {
   margin: 0;
 }
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: block;
+  float: right;
+}
+
+li a {
+  text-align: center;
+}
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#css): Don't use tag names for styling (except html and body)